plugins-ref/premium/powerpaste/powerpaste_support.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: default
+title: Support for the PowerPaste plugin
+title_nav: Support
```

Comment:
I'm not overly fond of this title. Would `Supported Functionality` work better or something? Makes me think of Zendesk Support.

Reply:
Maybe `Supported Systems` or `Supported Formats`? I agree `Support` is misleading here so we need something else.